shared/src/main/scala/Interpreter.scala

```diff
@@ -23,6 +24,7 @@ object Interpreter:
         then vyPrintln(ctx.peek)
       case Left(error) =>
         throw new Error(s"Error while executing $code: $error")
+  end execute
```

Comment:
Is this necessary?

Reply:
Scala linter. It just puts it there for stylistic consistency